.codeclimate.yml

```diff
@@ -0,0 +1,33 @@
+version: '2'
+checks:
+  file-lines:
+    enabled: true
+    config:
+      threshold: 300
+  method-lines:
+    enabled: false
+  method-count:
+    enabled: false
+  method-complexity:
+    enabled: false
+  file-lines:
+    enabled: false
+
+plugins:
+  duplication:
+    enabled: true
+  fixme:
+    enabled: true
+  sonar-python:
+    enabled: true
+    exclude_patterns:
+    - "**/settings.py"
+  pep8:
+    enabled: true
+
+exclude_patterns:
```

Comment:
when i ran codeclimate i got INFO log messages like 
```
INFO: Invalid character encountered in file '/code/search_gov_crawler/__pycache__/__init__.cpython-38.pyc' at line 3 for encoding UTF-8. Please fix file content or configure the encoding to be used using property 'sonar.sourceEncoding'.
```

I dont think we need to analyze the pycache folders, right?  maybe - `"**/__pycache__/"` or maybe `__pycache__/` could be added to the `exclude_patterns` list?